src/callbacks_step/analysis_dg2d_parallel.jl

```diff
@@ -161,7 +162,7 @@ function integrate_via_indices(func::Func, u,
                             normalize = normalize)
 
     # OBS! Global results are only calculated on MPI root, all other domains receive `nothing`
-    global_integral = MPI.Reduce!(Ref(local_integral), +, mpi_root(), mpi_comm())
+    global_integral = MPI.Reduce!(Ref(local_integral), reduce_vector_plus, mpi_root(), mpi_comm())
     if mpi_isroot()
         integral = convert(typeof(local_integral), global_integral[])
```

Comment:
If we do this, we may have to use a special handling if `local_integral isa Real`